src/components/AddressSearch/index.tsx

```diff
@@ -351,15 +252,15 @@ function AddressSearch({
                     address: CONST.YOUR_LOCATION_TEXT,
                     name: CONST.YOUR_LOCATION_TEXT,
                 };
-                onPress(location);
+                onPress?.(location);
             },
             (errorData) => {
                 if (!shouldTriggerGeolocationCallbacks.current) {
                     return;
                 }
 
                 setIsFetchingCurrentLocation(false);
-                setLocationErrorCode(errorData.code);
+                setLocationErrorCode(errorData.code as LocationErrorCodeType);
```

Comment:
Is it possible to update [this type](https://github.com/Expensify/App/blob/main/src/libs/getCurrentPosition/getCurrentPosition.types.ts#L15) instead of the assertion?

Reply:
Sure, done